packages/react-native-layout/src/components/flex/Flex.tsx

```diff
@@ -0,0 +1,61 @@
+import { View } from 'react-native'
+
+import type { FlexAlignProps, FlexJustifyProps, FlexProps } from './Flex.props'
+
+const Flex = ({ testID, children, style, flexStyle }: FlexProps) => {
```

Comment:
other inspiration, making a single `Flex` component with typed props you might need: 
https://docs.nativebase.io/flex

or making your components accept utility props to set flex:
https://docs.nativebase.io/utility-first

Reply:
Well, that's kind of how this was approached;

**FlexRow**
- Sets `flexDirection: "row"`
- Accepts all other flexStyle properties as optional.

**FlexRowAlign**
- Sets `flexDirection: "row"
- Requires `alignItems` to be passed
- Accepts all other flexStyle properties as optional.

You get the gist ;-)